scoop/src/main/java/com/lyft/scoop/ViewController.java

```diff
@@ -6,14 +6,32 @@
 
     private boolean attached;
     private Scoop scoop;
+    private View view;
 
-    public void attach(View view) {}
+    public final void attach(View view) {
+        this.view = view;
+        attachController();
+    }
+
+    public void attachController() {}
 
     protected final boolean attached() {
         return this.attached;
     }
 
-    public void detach(View view) {}
+    public final void detach(View view) {
+        detachController();
+        this.view = null;
+    }
+
+    public void detachController() {}
+
+    public View getView() {
+        if (!attached) {
+            throw new IllegalStateException("View accessed while ViewController is detached.");
+        }
+        return this.view;
+    }
 
     void setAttached(boolean attached) {
```

Comment:
With removing the ability for subclasses to override the attach functionality, we can delete this function and handle the boolean inside `attach` `detach`